### PR TITLE
[MBO-1456] [MBO 2.4.x] Add tab for uninstalled modules in Module manager

### PR DESCRIPTION
**Pull request by sowbiba**
image

image

Fix for #612

### CODE REVIEW

File: views/templates/admin/controllers/module_catalog/uninstalled-modules.html.twig [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+{% block content %}
+  <div class="row justify-content-center">
+    <div class="col-lg-10">
+      {% block catalog_categories_listing %}
+        <div class="module-short-list">
+          {% if modules is empty %}
+            <div class="modules-list module-list-empty">
+              <p>
+                {{ 'You do not have any uninstalled module.' | trans({}, 'Modules.Mbo.Modulescatalog') }}
```
Comment (intraordinaire) [inline views/templates/admin/controllers/module_catalog/uninstalled-modules.html.twig:35]: The translation of this one is missing ?

**Review by intraordinaire — COMMENTED** [review]
Just missing a translation, looking good otherwise 
Bravo, and thanks !

**Review by intraordinaire — APPROVED** [review]

**Comment by micka-fdz** [comment]
Hi @sowbiba, is this new tab only added for PS 1.7 ?

**Comment by sowbiba** [comment]
> Hi @sowbiba, is this new tab only added for PS 1.7 ?

Hi @micka-fdz 
On PS 8.0+, uninstalled modules appear in the Module Manager listing

**Comment by micka-fdz** [comment]
I know that @sowbiba, I'm just asking if this new tab is hidden on PS 8.

**Pull request merged**